diff --git a/build/boards/cv181x/sg2000_milkv_duos_glibc_arm64_sd/memmap.py b/build/boards/cv181x/sg2000_milkv_duos_glibc_arm64_sd/memmap.py
index d18d314ca..3fba97cc3 100644
--- a/build/boards/cv181x/sg2000_milkv_duos_glibc_arm64_sd/memmap.py
+++ b/build/boards/cv181x/sg2000_milkv_duos_glibc_arm64_sd/memmap.py
@@ -40,10 +40,10 @@ class MemoryMap:
     # =================
     # Multimedia buffer. Used by u-boot/kernel/FreeRTOS
     # =================
-    ION_SIZE = 170 * SIZE_1M
-    H26X_BITSTREAM_SIZE = 2 * SIZE_1M
+    ION_SIZE = 0 * SIZE_1M
+    H26X_BITSTREAM_SIZE = 0 * SIZE_1M
     H26X_ENC_BUFF_SIZE = 0
-    ISP_MEM_BASE_SIZE = 20 * SIZE_1M
+    ISP_MEM_BASE_SIZE = 0 * SIZE_1M
     FREERTOS_RESERVED_ION_SIZE = H26X_BITSTREAM_SIZE + H26X_ENC_BUFF_SIZE + ISP_MEM_BASE_SIZE
 
     # ION after FreeRTOS
